packages/capnpc-ts/src/generators.ts

```diff
@@ -325,11 +326,18 @@ export function generateStructFieldMethods(
 
       }
 
+      const listClassIdentifier = ts.createIdentifier(listClass);
+
       adopt = true;
       disown = true;
       get = ts.createCall(
-        ts.createPropertyAccess(
-          THIS, '_getList'), __, [offsetLiteral, ts.createIdentifier(listClass)]);
+        ts.createPropertyAccess(THIS, '_getList'),
+        __,
+        [offsetLiteral, listClassIdentifier]);
+      init = ts.createCall(
```

Comment:
Not sure if you were expecting some fancy git tricks, but my git-foo is not the best, I'm just gonna add your changes to my own.